dimagi/commcare-hq
# Datadog Metric Slicing

## Product Description


This related Formplayer PR affects metric `formplayer.metrics.timings`.
- Datadog tag `module_initiated_by` is renamed to `request_initiated_by`.

This PR adds the values `pagination` and `clickable_icon` to capture requests that are triggered by user action pagination and clickable icon.

![Screen Shot 2024-03-22 at 9 56 21 PM](https://github.com/dimagi/commcare-hq/assets/88759246/ba6a5347-7d1b-47a5-8c21-6bfa2cf02e46)



## Technical Summary

[Jira Ticket](https://dimagi.atlassian.net/browse/USH-4182)
[Related Formplayer Ticket](https://github.com/dimagi/formplayer/pull/1551)

See FP PR for reasoning of changing `module_initiated_by` tag.

## Feature Flag

no FF

## Safety Assurance

### Safety story

Will test on staging. Blast radius is contained to clickable icons, pagination, dynamic search, and case search validation.

### Automated test coverage


no automated tests

### QA Plan


QA to ensure no impact on clickable icons, pagination, dynamic search, and case search validation.
 [QA Ticket](https://dimagi.atlassian.net/browse/QA-6188?focusedCommentId=316724&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-316724)
### Rollback instructions




### Labels & Review
